docs/COMPILATION.MD

```diff
@@ -189,7 +189,34 @@ Once set up you can run the GUI interface from the terminal `./ccextractorGUI`
 
 ## Windows
 
-Open the windows/ccextractor.sln file with Visual Studio (2015 at least), and build it.
+**Using Visual Studio**
+
```

Comment:
Place a note: Following screenshots and steps are written on basis of visualstudio version: ???

Reply:
`Place a note` was instruction for you, in document you may write Note:
ALso instead of ??? you have to fill your visual studio version